tox.ini

```diff
@@ -1,5 +1,14 @@
 [tox]
-envlist=docs
+envlist=py3{4,5,6},docs
+skip_missing_interpreters=True
+
+[testenv]
+deps=-rrequirements/tests.txt
+commands=
+    isort --check-only --diff -p shaarli_client
+    pycodestyle
+    pydocstyle
+    py34,py35: pytest --pylint
```

Comment:
For reference:


    Python 3.6 is not (yet) supported by a released pylint version
    hence, pylint execution is controlled by a Tox condition